enterprise/admin/__init__.py

```diff
@@ -1268,6 +1269,22 @@ class EnterpriseGroupMembershipAdmin(admin.ModelAdmin):
         'pending_enterprise_customer_user',
     )
 
+    def get_queryset(self, request):
```

Comment:
[suggestion] could we add in a test for this?

Reply:
Updated as per suggestion